src/main/kotlin/com/criptext/mail/scenes/composer/ComposerController.kt

```diff
@@ -463,7 +473,7 @@ class ComposerController(private val model: ComposerModel,
 
         dataSourceController.getAllContacts()
         scene.observer = observer
-
+        observer.showStartGuideAttachments()
```

Comment:
Should not use observer here, you should call the scene to show the attachments guide and use the observer to let know the controller whatever the scene need it to know after or during showing of that guide.